extensions/azdata/src/azdata.ts

```diff
@@ -118,6 +118,7 @@ export class AzdataTool implements azdataExt.IAzdataApi {
 						replaceEngineSettings?: boolean,
 						workers?: number
 					},
+					engineVersion?: string,
```

Comment:
this should be a parameter within the args object

Reply:
I don't agree with that - and honestly I'm not even sure why we have the args object anyways. That's not how we do the other commands such as dc create. 

Name is another argument but it's at the top level which seems fine to me.

Reply:
True the args object can be eliminated completely but with it present it makes for confusing usage as to what's in the args and what is not. We should clean this up.